weather-app/app/components/Sidebar.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import React, { useState } from "react";
+import styles from "./Sidebar.module.scss";
+
```

Comment:
import { usePathname } from "next/navigation";

This hook can be used to get the current pathname.

Reply:
Thanks for telling me!
I modified it.